agent/vendor/github.com/aws/amazon-ecs-agent/ecs-agent/acs/session/task_stop_verification_ack

```diff
@@ -0,0 +1,84 @@
+package session
```

Comment:
Where does this file come from? Was it added in on accident?

Reply:
Oops, yeah that was an accident, forgot to `go mod tidy && go mod vendor` after removing it

Reply:
Fixed!